1397

```diff
@@ -0,0 +1,161 @@
+# Sudoku Solver
+
+###### Helpful Resources:
+
+[Sudoku Solver Geeks For Geeks](https://www.geeksforgeeks.org/sudoku-backtracking-7/)
+
+[Understanding Sudo solves in CPP](https://www.youtube.com/watch?v=lIOupNAafT0)
+
+[Introduction to Backtracking](https://www.youtube.com/watch?v=DKCbsiDBN6c)
+
+###### Solution Approach
+
+Firstly we implement a basic sudoku solver with four important points:
+
+1)We first have to find the empty positions for that we have the findEmptyLocation function
+2) To solve a sudoku we have to place a number at an empty position, '.' in this case such that this number is not present in the respective row, column or box, this is why we use the isSafe function
+3) If empty location found, we check if there exists a value that satisfies our required conditions of possible values that can be placed in that box, if for all boxes the number of values that needs to be exceeded exceeds 9 then that sudoku is unsolvable
+4) Here we are using a brute force to find the required solution via backtracking 
```

Comment:
Polish these 4 points a bit. You are on point but a bit vague. You can write something like this:

> In each iteration, we find an empty location. At the empty location, we place a value absent from the same row and column, and check if the current state of board can be solved. If it cannot be solved, we place a different value there and so on.